protocol-testing/src/test_runner.rs

```diff
@@ -61,42 +74,45 @@ impl TestRunner {
             .join("integration_test.tycho.yaml");
 
         let config = Self::parse_config(&config_yaml_path)?;
+        let terminal_width = termsize::get()
+            .map(|size| size.cols as usize - 35) // Remove length of log prefix
+            .unwrap_or(80);
 
         info!("Running {} tests ...\n", config.tests.len());
```

Comment:
the number of tests depends on the filter.. I tried to run with a filter for 1 test and still got the the log 
`Running 7 tests ...`

Reply:
same for the passed log
`Passed 7/7`